src/screens/Home.js

```diff
@@ -13,7 +11,12 @@ export default function Home() {
         subheading="Our monthly-ish in-person get togethers"
         text="Our meetups are handled over on Meetup.com. Join the group over there to get notifications about upcoming events."
         buttons={[
-          <a href="https://www.meetup.com/BendJS/" className="button">
+          <a
+            className="button"
+            href="https://www.meetup.com/BendJS/"
+            target="_blank"
```

Comment:
👏

Reply:
I'm typically against target="_blank" links just because it's an external site (https://css-tricks.com/use-target_blank/) but my care level isn't particularly high here and I'm clearly outvoted ;)

Reply:
@chriscoyier This is great feedback, and your article makes some really great points. I definitely lean towards embracing the platform and not messing with default browser behavior. With external links, websites open them in a new tab so commonly, I wonder what most users prefer or expect, and if there's any convergence in opinion.

I'm not personally attached to using `target="_blank"`, and I enjoy questioning decisions like this. It's currently just a habit, so I'm cool with leaving it out in future iterations.

Reply:
I was on the OG convention train, but those are great points, I'm all for the new future :rocket: It is interesting too with mobile browsers how you really have one view context so often a new tab or not it'll feel the same -- hitting the back button on the device just closes the new tab and goes back to the old so there's not much reason to even make a new tab (I don't think many mobile users are opening up their tab list to jump between tabs, they tend to have a linear single threaded flow)

Thanks for the article @chriscoyier 👍

I'm also surely biased since I'm often on my device spawning new tabs whether or not the page meant it to be that way. Then I get frustrated when I wish something was a link but it's really just an element with a click handler and can't be spawned in a new tab :P 

I'm sure you get more accurate analytic metrics too like knowing actual session time.

Reply:
@ctsstc All good points. I too find myself on mobile long-pressing every link to open in a new tab. I think this habit is mainly formed from not wanting the current page to reload fully if I go back to it, and I'm usually in the middle reading an article, but want to check out linked resources after I'm done reading. It's probably a very personal thing.